chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/endpoints/ChaosMonkeyRequestScopeRestEndpointDisabledIntTest.java

```diff
@@ -36,7 +36,7 @@
     classes = ChaosDemoApplication.class,
     webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @TestPropertySource("classpath:test-chaos-monkey-endpoints-disabled.properties")
-class ChaosMonkeyRequestScopeRestEndpointDisabledIntegration {
+class ChaosMonkeyRequestScopeRestEndpointDisabledIntTest {
```

Comment:
I would stick here with the longer name "...IntegrationTest" instead of "...IntTest"

Reply:
Where should I add myself in changelog?

Reply:
In the contributors list of the changelog. 
There is an example with octocat (`- https://github.com/octocat[@octocat]`), for you it would be:
`- https://github.com/anurag-rajawa[@anurag-rajawa]`